### DISCUSSION TITLE
Card Token Question

### DISCUSSION THREAD

**Discussion by jorgevasquezutec** [mercadopago/sdk-js#134] [General]
I'm trying to create a first payment and then make a subscription with the first month free and a different amount than the first payment. Currently it occurred to me to make the payment with the API and normal generates the payment; However, when I want to generate the subcription with the same cardToken, it says that it cannot be done. My question is what should be the correct flow to solve this solution?

**Comment by lucmantovani** [comment]
Hello @jorgevasquezutec!

Card tokens can be used only once
If you want to create a single payment and afterwards create a subscription, after creating the payment you should collect card data again and create another token

But there are also options for configuring a subscription with pending payments: https://www.mercadopago.com/developers/en/docs/subscriptions/integration-configuration/subscription-no-associated-plan

**Reply by jorgevasquezutec** [reply]
I understand, the subscription with pending payments allows me to create a subscription without entering the card_token_id , however if I want to charge at some point I would have to insert the payment method and ask the user for that information. Is there a way to generate two cardTokens from CardForm just entering the card information once and send them to the backend?

**Reply by lucmantovani** [reply]
No, there is not a way of generating two tokens at once
An alternative though is to save a Customer and Card and afterwards just request the security code

For these kind of questions about implementation, and since this space is intended for discussions about the SDKs, the most recommended channel is our support: https://www.mercadopago.com/developers/en/support/center/tickets
